lib/player.js

```diff
@@ -1065,6 +1080,8 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       payload.startTime = startTime;
     }
 
+    this.startTime_ = null;
```

Comment:
If you moved this to up before the `loading` event is fired, users could also call the event listener inside that event listener.
I don't know if that would be useful at all, but it might as well be possible.

Reply:
done